src/pymor/la/numpysolvers.py

```diff
@@ -481,15 +481,15 @@ def apply_inverse(matrix, U, options=None):
         else:
             if hasattr(matrix, 'factorization'):
                 for i, UU in enumerate(U):
-                    R[i] = matrix.factorization.solve(UU)
```

Comment:
Are you sure there is a problem? If `U.ndim == 2`, then `UU` should already be a one-dimensional array ...